manifests/g/Google/Chrome/131.0.6778.140/Google.Chrome.installer.yaml

```diff
@@ -0,0 +1,46 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Google.Chrome
+PackageVersion: 131.0.6778.140
+InstallerLocale: en-US
+InstallerType: msi
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- http
+- https
+- mailto
+- mms
+- tel
+- webcal
+FileExtensions:
+- htm
+- html
+- pdf
+- shtml
+- svg
+- webp
+- xht
+- xhtml
+ProductCode: '{D9596C6B-431E-3638-ACB7-B4B0D24D2D1B}'
+ReleaseDate: 2024-12-10
+AppsAndFeaturesEntries:
+- DisplayName: Google Chrome
+  Publisher: Google LLC

```

Comment:
`DisplayName` and `Publisher` are not needed since they match the DefaultLocale file

Reply:
https://github.com/russellbanks/Komac/commit/88cd84c84b09d16c2ca1d70d6974c920b47dd106